### DISCUSSION TITLE
problem and solved "[FATAL] listen udp 127.0.0.1:53: bind: permission denied"

### DISCUSSION THREAD

**Discussion by zhuyingda** [DNSCrypt/dnscrypt-proxy#2715] [Q&A]
I encountered the same issue as described in [this issue](https://github.com/DNSCrypt/dnscrypt-proxy/issues/108)

### Environment details: 
MacOS 13.7, Homebrew 4.3.18 

### problem detail and solved workaround
I installed dnscrypt-proxy using [homebrew](https://brew.sh/) and started the service with `sudo brew services start dnscrypt-proxy` , as port 53 requires root privileges. When checking the logs, I also saw multiple `[NOTICE] Network connectivity detected` and `[FATAL] listen udp 127.0.0.1:53: bind: permission denied` messages.
I resolved the issue by running `brew services stop dnscrypt-proxy` and then `sudo brew services restart dnscrypt-proxy`.

I believe the cause of the "permission denied" error was an existing dnscrypt-proxy process started by Homebrew, which conflicted with the new service start attempt.

I wish it would be helpful to others.

**Comment by LycorisWave** [comment]
choose another port, for example 1024
